software/webapp/src/test/java/brooklyn/entity/webapp/jboss/Wildfly8ServerDockerLiveTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package brooklyn.entity.webapp.jboss;
+
+import org.testng.annotations.Test;
+
+import brooklyn.entity.webapp.JavaWebAppSoftwareProcess;
+import brooklyn.location.Location;
+
+/**
+ * A simple test of installing+running Wildfly 8 on Docker, using various OS distros and versions.
+ */
+public class Wildfly8ServerDockerLiveTest extends JBossServerDockerLiveTest {
```

Comment:
As with the Wildfly AWS test, will this fail because the attributes aren't being set? In which case we should include a comment explaining why.

Could also parameterise the `super.doTest()` to say whether to assert those attributes are non-null.